src/main/java/se/sundsvall/chat/api/model/MessageRequest.java

```diff
@@ -0,0 +1,119 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a message within a conversation, including metadata and content.")
+public class MessageRequest {
+
+	@Schema(description = "The sequence number of the message in the conversation.", example = "1")
+	private String sequenceNumber;
+
+	@Schema(description = "The identifier of the message this message is replying to, if any.", example = "msg-12344")
+	private String inReplyTo;
+
+	@Schema(description = "The participant who created the message.")
+	private Participant createdBy;
+
+	@Schema(description = "The content of the message.", example = "Hello, how can I help you?")
+	private String content;
+
+	@ArraySchema(schema = @Schema(implementation = Participant.class, description = "The list of participants who have read the message."))
```

Comment:
"The list of participants who have read the message"

We can have other users than participants, that are reading a message.
(See my comment about the Participant-word in the ParticipantEntity class)